### PR TITLE
Added Moonrise-class Emergency Response Cruiser (Non-voteable)

### PR DESCRIPTION
**Pull request by LittleBuilderJane**
## About the PR 
Adds the Moonrise-class Cruiser.
**Screenshots**

Moonrise Screenshot
**Changelog**

:cl: Schiene
- add: Added a new non-votable map, Moonrise-class Emergency Response Cruiser.

### CODE REVIEW

**Comment by LittleBuilderJane** [comment]
The Picture is slightly outdated, the shuttle DOES have windows.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]
A bunch of pieces are going to detach immediately whenever grid splitting is coded.

**Comment by LittleBuilderJane** [comment]
> A bunch of pieces are going to detach immediately whenever grid splitting is coded.



> A bunch of pieces are going to detach immediately whenever grid splitting is coded.

Updated and fixed.

File: Resources/Prototypes/Maps/game.yml [hunk 1]
```diff
@@ -41,6 +41,10 @@
     prefixCreator: 'VG'
   mapPath: Maps/packedstation.yml
   minPlayers: 15
+  conditions:
+    - !type:HolidayMapCondition
+      inverted: true
+      holidays: [ "Christmas" ]
   overflowJobs:
```
Comment (metalgearsloth) [inline Resources/Prototypes/Maps/game.yml:48]: ?

File: Resources/Prototypes/Maps/game.yml [hunk 2]
```diff
@@ -86,3 +90,49 @@
...
+  availableJobs:
+    Bartender: [ 1, 1 ]
+    Captain: [ 1, 1 ]
+    ChiefEngineer: [ 1, 1 ]
+    StationEngineer: [ 1, 1 ]
+    ChiefMedicalOfficer: [ 1, 1 ]
+    MedicalDoctor: [ 1, 1 ]
+    ResearchDirector: [ 1, 1 ]
+    Botanist: [ 1, 1 ]
+
```
Comment (metalgearsloth) [inline Resources/Prototypes/Maps/game.yml:114]: Not a map

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Maps/game.yml (hunk 1 above)
Comment (LittleBuilderJane) [reply Resources/Prototypes/Maps/game.yml]: Dunno where that came from, but it's most definitely NOT my code. I didn't write that.
Removed it.

File: Resources/Prototypes/Maps/game.yml (hunk 2 above)
Comment (LittleBuilderJane) [reply Resources/Prototypes/Maps/game.yml]: That's from another branch of mine, dunno how it got there.
Removed.

**Comment by metalgearsloth** [comment]
Requires https://github.com/space-wizards/space-station-14/pull/6392

**Pull request merged**